src/Microsoft.IdentityModel.Protocol.Extensions/AudienceValidator.cs

```diff
@@ -31,19 +32,21 @@ internal class AudienceValidator
     {
         public static void Validate(IEnumerable<string> audiences, SecurityToken securityToken, TokenValidationParameters validationParameters)
         {
+            IdentityModelEventSource.Logger.WriteInformation("AudienceValidator.Validate: validating audience in the jwt token");
+
             if (audiences == null)
             {
-                throw new ArgumentNullException("audiences");
+                LogHelper.LogError(string.Format(CultureInfo.InvariantCulture, ErrorMessages.IDX10000, "AudienceValidator.Validate: audiences"), typeof(ArgumentNullException));
```

Comment:
Logging the parameter validations or null checks when it throws, is that really an issue? The program throws and exits anyways after this statement, I don't think it will fill in the logs with too much info.